### PR TITLE
Hasan/Pr 1872 lighthouse v11 accessibility issue

### PR DESCRIPTION
**Pull request by hasan-deriv**
Changes:

-   Fixed accessibility issues of deriv.com

### CODE REVIEW

**Review by meenakshi-deriv — DISMISSED** [review]

File: src/features/components/templates/navigation/payment-agent-nav/index.tsx [hunk 1]
```diff
@@ -86,11 +84,20 @@ const PaymentAgentAffiliateNav = ({ is_prime_page = false }: { is_prime_page?: b
...
+                            src={PrimeLogo}
+                            className={partners_nav_logo}
+                            alt="deriv prime logo"
+                        />
                     ) : (
-                        <Image src={PartnerNavLogo} className={partners_nav_logo} />
+                        <Image
+                            src={PartnerNavLogo}
+                            className={partners_nav_logo}
+                            alt="deriv partners logo"
```
Comment (meenakshi-deriv) [inline src/features/components/templates/navigation/payment-agent-nav/index.tsx:99]: @hasan-deriv shouldn't we localize all alt tags?

File: src/components/custom/_dhero-dmt5.tsx [hunk 2]
```diff
@@ -137,7 +137,7 @@ const DCommonBanner = () => {
             <StyledContainer jc="flex-start">
                 <ContentWrapperStyle is_rtl={is_rtl}>
                     <Content>
-                        <StyledTradingLogin src={DerivMT5Logo} />
+                        <StyledTradingLogin src={DerivMT5Logo} alt="Deriv MT5 logo" />
```
Comment (mayuran-deriv) [inline src/components/custom/_dhero-dmt5.tsx:140]: i think we need translation for alt

File: src/features/pages/home/live-pricing/market-tabs/market-tab.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ import useRegion from 'components/hooks/use-region'
 const MarketTab = () => {
     const [selected_market, setSelectedMarket] = useState<TAvailableLiveMarkets>('fx')
     const [tab_name, setTabName] = useState('Forex')
-    const [linkToMarketPage, setLinkToMarketPage] = useState('/markets/forex/')
+    const [linkToMarketPage, setLinkToMarketPage] = useState<`/${string}`>('/markets/forex/')
```
Comment (mayuran-deriv) [inline src/features/pages/home/live-pricing/market-tabs/market-tab.tsx:16]: this been used in several places , can we add this type in a common place

Comment (mayuran-deriv) [reply src/features/pages/home/live-pricing/market-tabs/market-tab.tsx:16]: TpathString = `/${string}`

File: src/pages/payment-methods/_expanded-list.tsx [hunk 4]
```diff
@@ -132,6 +131,7 @@ const ExpandList = ({ payment_data, is_fiat_onramp, locale }: PaymentProps) => {
     return (
         <>
             <Tr is_expanded={is_expanded}>
+                <th scope="row">dfasdf</th>
```
Comment (mayuran-deriv) [inline src/pages/payment-methods/_expanded-list.tsx:134]: dfasdf ???

File: src/pages/trade-types/components/_tabs.tsx [hunk 5]
```diff
@@ -175,15 +185,18 @@ const Tabs = ({
...
                         return (
                             <>
                                 <TabButton
                                     className="side-tab__button"
                                     role="tab"
+                                    id={tabID}
+                                    aria-controls={panelID}
                                     selected={selected_tab === index}
                                     aria-selected={selected_tab === index ? 'true' : 'false'}
+                                    aria-expanded={selected_tab === index ? 'true' : 'false'}
```
Comment (mayuran-deriv) [inline src/pages/trade-types/components/_tabs.tsx:199]:
```suggestion
                                    aria-expanded={selected_tab === index}
```

File: src/pages/payment-methods/_expanded-list.tsx (hunk 4 above)
Comment (hasan-deriv) [reply src/pages/payment-methods/_expanded-list.tsx:134]: It's special features 😆

**Pull request closed without merging**